fuzzers/lafintel/fuzzer.py

```diff
@@ -29,7 +29,9 @@ def prepare_build_environment():
     # (see https://github.com/php/php-src/blob/master/Zend/zend_cpuinfo.h).
     # It is not supported by clang-3.8, so we define the MACRO below
     # to replace any __builtin_cpu_supports() with 0, i.e., not supported
-    cflags = ['-O3', '-fPIC', '-D__builtin_cpu_supports\\(x\\)=0']
+    cflags = ['-O3', '-fPIC']
```

Comment:
ah, can you remove -O3 and utils.set_no_sanitizer_compilation_flags() call from above.  this is not needed anymore after my yesterday fix.
also, can you do same fix in aflplusplus_noalloc which was checked in after my fix.